examples/simulated/plot_class_distinctiveness.py

```diff
@@ -0,0 +1,108 @@
+"""
+=====================================================================
+Select best class-separated SPD matrix dataset on a manifold
+=====================================================================
+Selecting SPD matrix dataset with better separation between two classes on a manifold.
+This could be used as the metric for selecting the best frequency band or time window among multiple choices.
+"""
+# Authors: Maria Sayu Yamamoto <maria-sayu.yamamoto@universite-paris-saclay.fr>
+#
+# License: BSD (3-clause)
```

Comment:
Indeed, these two examples have much in common. I will try to incorporate the class_dis example into `plot_toy_classification` !

Reply:
I have incorporated those two examples, so please confirm the new example.
Originally, in `plot_class_distinctiveness.py`, the objective of the example was to show the differences in distribution between the best dataset selected by the class_dis function and the other datasets.
On the other hand, in the integrated example, I changed the objective to show how the class_dis value varies as the delta (a parameter for controlling the distance between centers) increases, in order to integrate the class_dis example by respecting the original `plot_toy_classification.py` as much as possible.

Reply:
But, this new example shows a very obvious class_dis result as the dispersion of the Gaussian distribution is fixed, and only the distance between centroids is controlled. Could I change the dispersion as well? Or is it better to leave the example simple rather than increase complexity?